proposals/lip-0048.md

```diff
@@ -25,15 +25,34 @@ This LIP defines the fee system in a modular way, as currently [used in the Lisk
 
 ## Rationale
 
+The fee structure presented here serves to allow defining sufficient cost to manage state growth and prevent transaction pool spamming. In principle, any transaction resulting in a state change should incur a fee. This reasoning is implemented in the `payFee` method, which can be invoked by other modules.
+
 ### Fee Token ID
 
-Each chain can configure the token used to pay fees. On the Lisk mainchain, the token used for transaction fees is the LSK token.
+Each chain can configure the token used to pay fees, as defined by `TOKEN_ID_FEE`. On the Lisk mainchain, the token used for transaction fees is the LSK token.
 
 ### Minimum Fee per Transaction
 
-As introduced in [LIP 0013][lip-0013], all transactions must have a fee greater or equal to a minimum fee (which can be zero). The minimum fee is computed from the transaction size. Chains can configure their minimum fee per byte, with a possibility to define a period of `MAX_BLOCK_HEIGHT_ZERO_FEE_PER_BYTE` blocks after the genesis block when the minimum fee per byte is set to zero.
+As introduced in [LIP 0013][lip-0013], all transactions must have a fee greater or equal to a minimum fee, which can be set to zero. For a transaction `trs`, the minimum fee is determined by its size and the proportionality constant `MIN_FEE_PER_BYTE`: 
+
+```python
+trsSize = length(encodeTransaction(trs))
+minFee = MIN_FEE_PER_BYTE * trsSize
+```
+
+Note that chains can freely configure their `MIN_FEE_PER_BYTE`, with a possibility to define a period of `MAX_BLOCK_HEIGHT_ZERO_FEE_PER_BYTE` blocks after the genesis block when it is set to zero.
```

Comment:
We should add a rationale for why we allow for this. Specifically, I think it has to do with using LSK as the fee token in sidechains.

Reply:
Noted.